src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -108,12 +111,13 @@ export const RichTextEditor: FC<RichTextEditorProps> = ({
             <div data-test-id="rich-text-editor" className="tw-relative tw-w-full" ref={editorRef}>
                 <Plate
                     id={editorId}
-                    initialValue={parseRawValue(initialValue)}
+                    initialValue={parseRawValue(initialValue, config)}
                     onChange={onChange}
                     editableProps={editableProps}
-                    plugins={getEditorConfig()}
+                    plugins={getEditorConfig(config)}
                 >
                     <Toolbar editorId={editorId} actions={actions} editorWidth={editorWidth} />
+                    {config === EditorConfigType.ANNOTATIONS && <MentionCombobox items={mentionable} />}
```

Comment:
Should we not control that via the editorActions as well? 🤔 
Instead of introducing this new config "EditorConfigType"

Reply:
The way the Slate and Plate works is like this and cant be controlled over Actions. You need mention combobox and when you have it every RTE is dependent and mentions working in all. This is not a toolbar option, there is no action for mentions the actions are accepted by toolbar and if there is a missing action you can use it in a toolbar. It is not dependent on toolbar.